test/e2e/data/infrastructure-oci/v1beta1/cluster-template-cluster-class.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: cluster.x-k8s.io/v1beta1
+kind: Cluster
```

Comment:
we should remove this file from this PR

Reply:
similar to below files

Reply:
do you think it is worthwhile to add a task for adding e2e test for this feature?